index.html

```diff
@@ -224,6 +243,7 @@ <h3>
             USVString title;
             USVString text;
             USVString url;
+            FileList files;
```

Comment:
Also, I wonder if `FileList` should be considered deprecated, due to the comment [here](https://www.w3.org/TR/FileAPI/#filelist-section):

> Note: The FileList interface should be considered "at risk" since the general trend on the Web Platform is to replace such interfaces with the Array platform object in ECMAScript [ECMA-262].

Reply:
The link in the IDL is now fixed.